source/includes/fact-connect-auth-info.rst

```diff
@@ -37,20 +37,15 @@ method from the tabs below for specific instructions:
          authentication mechanism, see :manual:`SCRAM
          </manual/core/security-scram/>`.
 
-     - id: x509
-       name: X.509
-       content: |
-         *Not Available in Compass Community Edition*
-
-         Select :guilabel:`X.509` if the ``mongod`` instance uses
-         :manual:`X.509 </core/security-x.509/>` as its authentication
-         mechanism. If selected, you must provide the
-         :guilabel:`Username` to authenticate the user.
-
      - id: kerberos
        name: Kerberos
        content: |
-         *Not Available in Compass Community Edition*
+
+         .. admonition:: Not Available in Compass Community Edition
```

Comment:
Do we have standards on color usage for different admonitions? I would expect this admonition to be yellow or red. Green seems like it'd be for a helpful tip in using the software? 

Also, the title of the admonition is redundant with the content of the admonition. The title is: "NOT AVAILABLE IN COMPASS COMMUNITY EDITION" and then in the admonition we have "...not available in Compass Community Edition."

Can we shorten the title? Do we have standards on what the titles are? I.e., "Note" "Warning" etc. etc. Are titles required to use admonitions?